### PR TITLE
Adding capture option

### PR DESCRIPTION
**Pull request by antonioperic**

### CODE REVIEW

**Comment by antonioperic** [comment]
@kayladnls can we merge this please?

**Comment by kayladnls** [comment]
@antonioperic looks good. I'll be happy to merge, but can we get some tests to cover the new functionality, please?

**Comment by antonioperic** [comment]
@kayladnls I promise I will add test, but now I need it for completing project. If you can merge I will deliver test until end of week. :)

**Comment by kayladnls** [comment]
I'm sorry, I really can't merge without a test. I'll see if I can get the tests written this evening and merge it in the morning.

**Comment by antonioperic** [comment]
@kayladnls ok, if you can do that it will be awesome

**Comment by antonioperic** [comment]
@kayladnls did you manage to work on this?
thnx

**Comment by antonioperic** [comment]
@kayladnls hi, I added Cpautre, Annul, Credit request and covered with tests, please merge this, it is very important for one of project I am working on. If need something to change, please let me know

File: src/Gateway.php [hunk 1]
```diff
@@ -56,4 +56,19 @@ public function completePurchase(array $parameters = array())
     {
         return $this->createRequest('\Omnipay\Netaxept\Message\CompletePurchaseRequest', $parameters);
     }
+
+    public function capture(array $parameters = array())
+    {
+        return $this->createRequest('\Omnipay\Netaxept\Message\CaptureRequest', $parameters);
+    }
+
+    public function annul(array $parameters = array())
```
Comment (kayladnls) [inline src/Gateway.php]: In order to stay in line with the rest of Omnipay, this would need to be `void()`

**Comment by kayladnls** [comment]
@antonioperic This looks great, and that you very much for all the work on it. Just that one verbiage change on the Annul request. I'll merge it as soon as that's done.

**Comment by antonioperic** [comment]
@kayladnls  i am not sure I am following you

**Comment by kayladnls** [comment]
Omnipay keeps a consistent API across all gateways. So, what netaxept calls `annul`, we would call `void`. You would need to update that function on this gateway to reflect the consistent api. 

https://github.com/thephpleague/omnipay-netaxept/pull/4#discussion_r29942527

**Comment by antonioperic** [comment]
@kayladnls is now ok?

**Comment by kayladnls** [comment]
Yup, waiting on Travis and I'll tag a new minor.

**Comment by antonioperic** [comment]
thnx

**Pull request merged**